src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -2865,26 +2865,51 @@ The first mechanism is store file refresher which is introduced in HBase-1.0+. S
 
+[[async.wal.replication.meta]]
 === Region replication for META table’s region
-Currently, Async WAL Replication is not done for the META table’s WAL. The meta table’s secondary replicas still refreshes themselves from the persistent store files. Hence the `hbase.regionserver.meta.storefile.refresh.period` needs to be set to a certain non-zero value for refreshing the meta store files. Note that this configuration is configured differently than
-`hbase.regionserver.storefile.refresh.period`.
+Up until hbase-2.4.0, Async WAL Replication did not work for the META table’s WAL. The meta table’s secondary replicas refreshed themselves from the persistent store files every `hbase.regionserver.meta.storefile.refresh.period`,
+(a non-zero value). Note how the META replication period is distinct from the user-space `hbase.regionserver.storefile.refresh.period` value.
+
+Async WAL replication for META is a new feature in 2.4.0 still under active development. Use with caution.
+Set `hbase.region.replica.replication.catalog.enabled` to enable async WAL Replication for META region replicas.
+Its off by default.
+
+Regards the META replicas count, up to hbase-2.4.0, you would set the special property 'hbase.meta.replica.count'.
+Now you can alter the META table as you would a user-space table (if `hbase.meta.replica.count` is set, it
+will take precedent over what is set for replica count in the META table updating META replica count to
+match).
+
+==== Load Balancing META table load ====
+
+hbase-2.4.0 adds a new client-side `LoadBalance` mode. When enabled client-side, clients will try to read META replicas first before falling back on the primary. Before this,
+the lookup mode -- now named `HedgedRead` -- had clients read the primary and if no response after a configurable amount of time had elapsed, it would start up reads against the replicas.
+The new 'LoadBalance' mode helps alleviate hotspotting on the META table distributing the META read load.
+
+To enable the meta replica locator's load balance mode, please set the following configuration at on the client-side (only): set 'hbase.locator.meta.replicas.mode' to "LoadBalance".
+Valid options for this configuration are `None`, `HedgedRead`, and `LoadBalance`. Option parse is case insensitive.
+The default mode is `None` (which falls through to `HedgedRead`, the current default).  Please do not put this configuration in any hbase server's configuration, master or region server.
```

Comment:
"please do not" is this language strong enough? What happens if i ship out this configuration accidentally? How badly will we muck up meta? Should the the system proactively defend against this configuration?

Reply:
Change it to DO NOT. A follow-on will add code to defend against this being set server side.

Reply:
Added HBASE-25294 to add code defense against the LoadBalance option being enabled for the Master-hosted client.